.github/workflows/all.yml

```diff
@@ -1,28 +1,38 @@
 name: Build and test the aws jslib against a Localstack instance
-on: push
+on:
+    # Enables running the workflow manually from the Actions tab
+    workflow_dispatch:
+
+    push:
+        branches:
+            - main
+        tags:
+            - v*
+
+    pull_request:
 
 jobs:
-  test:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
```

Comment:
Since, in any case, you've touched this file, I recommend you update `actions/checkout@v2` (and `actions/setup-go@v2`) to a newer version since it's deprecated. 

See [an output](https://github.com/grafana/k6-jslib-aws/actions/runs/6902488750) of the jobs:
```
The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2, actions/setup-go@v2. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/
```

Reply:
I forgot to do that indeed, good catch 🎉